? config_office/config.log
? config_office/config.parms
? config_office/config.status
? config_office/findhome.class
? config_office/findhome.java
? config_office/set_soenv
? config_office/warn
? libtextcat/unxlngx6.pro
? libxmlsec/unxlngx6.pro
? libxslt/unxlngx6.pro
Index: config_office/bootstrap.1
===================================================================
RCS file: /cvs/tools/config_office/bootstrap.1,v
retrieving revision 1.21
diff -u -u -r1.21 bootstrap.1
--- config_office/bootstrap.1	26 Jun 2007 17:48:08 -0000	1.21
+++ config_office/bootstrap.1	11 Apr 2008 07:54:07 -0000
@@ -36,7 +36,8 @@
             $GNUMAKE distclean || exit
         fi
 
-        ./configure $DMAKE_CONF || exit
+        ./configure $DMAKE_CONF --host=$(DEB_HOST_GNU_TYPE) \
+		--build=$(DEB_BUILD_GNU_TYPE) || exit
 
         ## invoke the gnu make command set by configure.
         $GNUMAKE || exit
Index: config_office/set_soenv.in
===================================================================
RCS file: /cvs/tools/config_office/set_soenv.in,v
retrieving revision 1.168.4.1
diff -u -u -r1.168.4.1 set_soenv.in
--- config_office/set_soenv.in	23 Jan 2008 17:07:28 -0000	1.168.4.1
+++ config_office/set_soenv.in	11 Apr 2008 07:54:08 -0000
@@ -1602,6 +1602,8 @@
 CreateFileHeader( *OUT_SH, $UPD, $platform, "sh/bash/ksh", "#" );
 CreateFileHeader( *OUT_BAT, $UPD, $platform, "4nt", "rem" ) if defined($outfile_bat);
 
+ToFile( "DEB_BUILD_GNU_TYPE",`dpkg-architecture -qDEB_BUILD_GNU_TYPE`, "e" );
+ToFile( "DEB_HOST_GNU_TYPE", `dpkg-architecture -qDEB_HOST_GNU_TYPE`,  "e" );
 ToFile( "SRC_ROOT",	     $SRC_ROOT,          "e" );
 ToFile( "SHELL",             $SHELL,             "e" );
 ToFile( "JAVA_HOME",         $JAVA_HOME,         "e" );
Index: berkeleydb/makefile.mk
===================================================================
RCS file: /cvs/external/berkeleydb/makefile.mk,v
retrieving revision 1.43.2.1
diff -u -u -r1.43.2.1 makefile.mk
--- berkeleydb/makefile.mk	2 Jan 2008 17:30:31 -0000	1.43.2.1
+++ berkeleydb/makefile.mk	11 Apr 2008 07:54:08 -0000
@@ -103,7 +103,7 @@
 #relative to CONFIGURE_DIR
 CONFIGURE_ACTION= \
 	..$/dist$/configure
-CONFIGURE_FLAGS=--disable-cxx --enable-dynamic --enable-shared --enable-compat185
+CONFIGURE_FLAGS=--disable-cxx --enable-dynamic --enable-shared --enable-compat185 --host=$(DEB_HOST_GNU_TYPE) --build=$(DEB_BUILD_GNU_TYPE)
 
 # just pass ARCH_FLAGS to native build
 CFLAGS+:=$(ARCH_FLAGS)
Index: curl/makefile.mk
===================================================================
RCS file: /cvs/external/curl/makefile.mk,v
retrieving revision 1.22
diff -u -u -r1.22 makefile.mk
--- curl/makefile.mk	21 Sep 2007 09:26:36 -0000	1.22
+++ curl/makefile.mk	11 Apr 2008 07:54:08 -0000
@@ -70,7 +70,7 @@
 CONFIGURE_DIR=.$/
 #relative to CONFIGURE_DIR
 CONFIGURE_ACTION=.$/configure
-CONFIGURE_FLAGS= --without-ssl --without-libidn --enable-ftp --enable-ipv6 --enable-http --disable-gopher --disable-file --disable-ldap --disable-telnet --disable-dict --disable-static CPPFLAGS="$(curl_CFLAGS)"  LDFLAGS="$(curl_LDFLAGS)"
+CONFIGURE_FLAGS= --without-ssl --without-libidn --enable-ftp --enable-ipv6 --enable-http --disable-gopher --disable-file --disable-ldap --disable-telnet --disable-dict --disable-static --host=$(DEB_HOST_GNU_TYPE) --build=$(DEB_BUILD_GNU_TYPE) CPPFLAGS="$(curl_CFLAGS)"  LDFLAGS="$(curl_LDFLAGS)"
 
 BUILD_DIR=$(CONFIGURE_DIR)$/lib
 .IF "$(OS)"=="IRIX"
Index: epm/makefile.mk
===================================================================
RCS file: /cvs/external/epm/makefile.mk,v
retrieving revision 1.4.8.1
diff -u -u -r1.4.8.1 makefile.mk
--- epm/makefile.mk	2 Jan 2008 17:30:11 -0000	1.4.8.1
+++ epm/makefile.mk	11 Apr 2008 07:54:08 -0000
@@ -50,7 +50,7 @@
 .IF "$(GUI)"=="UNX" && "$(BUILD_EPM)" != "NO"
 
 CONFIGURE_ACTION=.$/configure
-CONFIGURE_FLAGS=--disable-fltk
+CONFIGURE_FLAGS=--disable-fltk --host=$(DEB_HOST_GNU_TYPE) --build=$(DEB_BUILD_GNU_TYPE)
 .IF "$(OS)"=="MACOSX"
 CONFIGURE_FLAGS+=CFLAGS="$(EXTRA_CFLAGS)" LDFLAGS="$(EXTRA_LINKFLAGS)"
 .ENDIF
Index: fondu/makefile.mk
===================================================================
RCS file: /cvs/external/fondu/makefile.mk,v
retrieving revision 1.4
diff -u -u -r1.4 makefile.mk
--- fondu/makefile.mk	18 Jul 2007 07:17:48 -0000	1.4
+++ fondu/makefile.mk	11 Apr 2008 07:54:08 -0000
@@ -56,7 +56,7 @@
 #relative to CONFIGURE_DIR
 CONFIGURE_ACTION=configure
 
-CONFIGURE_FLAGS=
+CONFIGURE_FLAGS= --host=$(DEB_HOST_GNU_TYPE) --build=$(DEB_BUILD_GNU_TYPE)
 
 BUILD_DIR=$(CONFIGURE_DIR)
 
Index: freetype/makefile.mk
===================================================================
RCS file: /cvs/external/freetype/makefile.mk,v
retrieving revision 1.24
diff -u -u -r1.24 makefile.mk
--- freetype/makefile.mk	3 Aug 2007 13:56:02 -0000	1.24
+++ freetype/makefile.mk	11 Apr 2008 07:54:08 -0000
@@ -62,7 +62,7 @@
 #relative to CONFIGURE_DIR
 .IF "$(OS)"=="MACOSX"
 CONFIGURE_ACTION=./configure
-CONFIGURE_FLAGS=--with-pic --without-zlib
+CONFIGURE_FLAGS=--with-pic --without-zlib --host=$(DEB_HOST_GNU_TYPE) --build=$(DEB_BUILD_GNU_TYPE)
 .ELSE
 CONFIGURE_ACTION=$(GNUMAKE)
 CONFIGURE_FLAGS=setup CFG="--with-pic --without-zlib"
Index: icu/makefile.mk
===================================================================
RCS file: /cvs/external/icu/makefile.mk,v
retrieving revision 1.37.16.1
diff -u -u -r1.37.16.1 makefile.mk
--- icu/makefile.mk	2 Jan 2008 17:31:44 -0000	1.37.16.1
+++ icu/makefile.mk	11 Apr 2008 07:54:08 -0000
@@ -99,7 +99,7 @@
 
 CONFIGURE_DIR=source
 
-CONFIGURE_ACTION=sh -c 'CFLAGS="$(icu_CFLAGS)" CXXFLAGS="$(icu_CXXFLAGS)" LDFLAGS="$(icu_LDFLAGS) $(LDFLAGSADD)" ./configure --enable-layout --enable-static --enable-shared=yes $(DISABLE_64BIT)'
+CONFIGURE_ACTION=sh -c 'CFLAGS="$(icu_CFLAGS)" CXXFLAGS="$(icu_CXXFLAGS)" LDFLAGS="$(icu_LDFLAGS) $(LDFLAGSADD)" ./configure --host=$(DEB_HOST_GNU_TYPE) --build=$(DEB_BUILD_GNU_TYPE) --enable-layout --enable-static --enable-shared=yes $(DISABLE_64BIT)'
 
 #CONFIGURE_FLAGS=--enable-layout --enable-static --enable-shared=yes --enable-64bit-libs=no
 CONFIGURE_FLAGS=
Index: libtextcat/makefile.mk
===================================================================
RCS file: /cvs/external/libtextcat/makefile.mk,v
retrieving revision 1.5.2.1
diff -u -u -r1.5.2.1 makefile.mk
--- libtextcat/makefile.mk	2 Jan 2008 17:33:39 -0000	1.5.2.1
+++ libtextcat/makefile.mk	11 Apr 2008 07:54:08 -0000
@@ -62,7 +62,7 @@
 
 #relative to CONFIGURE_DIR
 CONFIGURE_ACTION=configure CFLAGS="$(EXTRA_CFLAGS)"
-CONFIGURE_FLAGS=
+CONFIGURE_FLAGS= --host=$(DEB_HOST_GNU_TYPE) --build=$(DEB_BUILD_GNU_TYPE)
 
 BUILD_ACTION=make
 
Index: libxml2/makefile.mk
===================================================================
RCS file: /cvs/external/libxml2/makefile.mk,v
retrieving revision 1.17.18.1
diff -u -u -r1.17.18.1 makefile.mk
--- libxml2/makefile.mk	2 Jan 2008 17:30:52 -0000	1.17.18.1
+++ libxml2/makefile.mk	11 Apr 2008 07:54:08 -0000
@@ -93,7 +93,7 @@
 
 CONFIGURE_DIR=
 CONFIGURE_ACTION=.$/configure
-CONFIGURE_FLAGS=--enable-ipv6=no --without-python --enable-static=no --with-sax1=yes ADDCFLAGS="$(xml2_CFLAGS)" CFLAGS="$(EXTRA_CFLAGS)" LDFLAGS="$(xml2_LDFLAGS) $(EXTRA_LINKFLAGS)"
+CONFIGURE_FLAGS=--enable-ipv6=no --without-python --enable-static=no --with-sax1=yes --host=$(DEB_HOST_GNU_TYPE) --build=$(DEB_BUILD_GNU_TYPE) ADDCFLAGS="$(xml2_CFLAGS)" CFLAGS="$(EXTRA_CFLAGS)" LDFLAGS="$(xml2_LDFLAGS) $(EXTRA_LINKFLAGS)"
 BUILD_ACTION=$(GNUMAKE)
 BUILD_FLAGS+= -j$(EXTMAXPROCESS)
 BUILD_DIR=$(CONFIGURE_DIR)
Index: libxmlsec/makefile.mk
===================================================================
RCS file: /cvs/external/libxmlsec/makefile.mk,v
retrieving revision 1.15.6.1
diff -u -u -r1.15.6.1 makefile.mk
--- libxmlsec/makefile.mk	2 Jan 2008 17:29:48 -0000	1.15.6.1
+++ libxmlsec/makefile.mk	11 Apr 2008 07:54:08 -0000
@@ -139,7 +139,7 @@
 CONFIGURE_DIR=
 #CONFIGURE_ACTION=chmod 777 libxml2-config && .$/configure CFLAGS="$(xmlsec_CFLAGS)" CPPFLAGS="$(xmlsec_CPPFLAGS)" LDFLAGS="$(xmlsec_LDFLAGS)"
 CONFIGURE_ACTION=chmod 777 libxml2-config && .$/configure ADDCFLAGS="$(xmlsec_CFLAGS)" CPPFLAGS="$(xmlsec_CPPFLAGS)"
-CONFIGURE_FLAGS=--with-libxslt=no --with-openssl=no --with-gnutls=no LIBXML2LIB="$(LIBXML2LIB)" ZLIB3RDLIB=$(ZLIB3RDLIB)
+CONFIGURE_FLAGS=--with-libxslt=no --with-openssl=no --with-gnutls=no --host=$(DEB_HOST_GNU_TYPE) --build=$(DEB_BUILD_GNU_TYPE) LIBXML2LIB="$(LIBXML2LIB)" ZLIB3RDLIB=$(ZLIB3RDLIB)
 # system-mozilla needs pkgconfig to get the information about nss
 # FIXME: This also will enable pkg-config usage for libxml2. It *seems*
 # that the internal headers still are used when they are there but....
Index: libxslt/makefile.mk
===================================================================
RCS file: /cvs/external/libxslt/makefile.mk,v
retrieving revision 1.10.6.1
diff -u -u -r1.10.6.1 makefile.mk
--- libxslt/makefile.mk	2 Jan 2008 17:28:49 -0000	1.10.6.1
+++ libxslt/makefile.mk	11 Apr 2008 07:54:08 -0000
@@ -100,7 +100,7 @@
 .ENDIF                  # "$(COMNAME)"=="sunpro5"
 CONFIGURE_DIR=
 CONFIGURE_ACTION=chmod 777 libxml2-config && .$/configure
-CONFIGURE_FLAGS=--enable-ipv6=no --without-crypto --without-python --enable-static=no --with-sax1=yes CFLAGS="$(xslt_CFLAGS)" LIBXML2LIB=$(LIBXML2LIB) ZLIB3RDLIB=$(ZLIB3RDLIB)
+CONFIGURE_FLAGS=--enable-ipv6=no --without-crypto --without-python --enable-static=no --with-sax1=yes --host=$(DEB_HOST_GNU_TYPE) --build=$(DEB_BUILD_GNU_TYPE) CFLAGS="$(xslt_CFLAGS)" LIBXML2LIB=$(LIBXML2LIB) ZLIB3RDLIB=$(ZLIB3RDLIB)
 BUILD_ACTION=$(GNUMAKE)
 BUILD_FLAGS+= -j$(EXTMAXPROCESS)
 BUILD_DIR=$(CONFIGURE_DIR)
Index: portaudio/makefile.mk
===================================================================
RCS file: /cvs/external/portaudio/Attic/makefile.mk,v
retrieving revision 1.5
diff -u -u -r1.5 makefile.mk
--- portaudio/makefile.mk	8 Sep 2005 16:30:55 -0000	1.5
+++ portaudio/makefile.mk	11 Apr 2008 07:54:08 -0000
@@ -53,7 +53,7 @@
 CONFIGURE_DIR=
 
 CONFIGURE_ACTION=./configure
-CONFIGURE_FLAGS=
+CONFIGURE_FLAGS= --host=$(DEB_HOST_GNU_TYPE) --build=$(DEB_BUILD_GNU_TYPE)
 BUILD_DIR=$(CONFIGURE_DIR)
 BUILD_ACTION=$(GNUMAKE)
 
Index: python/makefile.mk
===================================================================
RCS file: /cvs/external/python/makefile.mk,v
retrieving revision 1.35
diff -u -u -r1.35 makefile.mk
--- python/makefile.mk	6 Nov 2007 15:58:43 -0000	1.35
+++ python/makefile.mk	11 Apr 2008 07:54:08 -0000
@@ -115,7 +115,7 @@
 .ENDIF
 
 
-CONFIGURE_ACTION=./configure --prefix=$(MYCWD)/python-inst --enable-shared CFLAGS="$(python_CFLAGS)" LDFLAGS="$(python_LDFLAGS)"
+CONFIGURE_ACTION=./configure --prefix=$(MYCWD)/python-inst --host=$(DEB_HOST_GNU_TYPE) --build=$(DEB_BUILD_GNU_TYPE) --enable-shared CFLAGS="$(python_CFLAGS)" LDFLAGS="$(python_LDFLAGS)"
 .IF "$(OS)$(CPU)" == "SOLARISI"
 CONFIGURE_ACTION += --disable-ipv6
 .ENDIF
Index: sndfile/makefile.mk
===================================================================
RCS file: /cvs/external/sndfile/Attic/makefile.mk,v
retrieving revision 1.9
diff -u -u -r1.9 makefile.mk
--- sndfile/makefile.mk	26 Mar 2007 12:22:58 -0000	1.9
+++ sndfile/makefile.mk	11 Apr 2008 07:54:08 -0000
@@ -63,7 +63,7 @@
 .EXPORT: LDFLAGS
 .ENDIF
 
-CONFIGURE_ACTION=./configure --with-pic
+CONFIGURE_ACTION=./configure --host=$(DEB_HOST_GNU_TYPE) --build=$(DEB_BUILD_GNU_TYPE) --with-pic
 
 BUILD_DIR=$(CONFIGURE_DIR)
 BUILD_ACTION=$(GNUMAKE)
